apis/Google.Cloud.Bigtable.Admin.V2/Google.Cloud.Bigtable.Admin.V2/BigtableInstanceAdminClientPartial.cs

```diff
@@ -0,0 +1,55 @@
+﻿// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Partial classes to support IAM policy versions
+
+using Google.Api.Gax.Grpc;
+using Google.Cloud.Iam.V1;
+
+namespace Google.Cloud.Bigtable.Admin.V2
+{
+    public partial class BigtableInstanceAdminSettings
+    {
+        /// <summary>
+        /// The version of IAM policies that the client will indicate that it supports in RPCs.
+        /// This defaults to a null value, meaning that requests will not be modified at all.
+        /// If a value is already present in a request, it is not overwritten.
```

Comment:
How does this get flipped between `null`, `1` or `3` version values?

Reply:
It's a property - a user just sets it on an instance: `settings.SupportedIamPolicyVersion = 3;`

Reply:
If they're using IAM v1, yes. But I'm not expecting that to be the common way of accessing IAM by that time.

Reply:
Fair point!